src/weapon.py

```diff
@@ -181,9 +196,8 @@ def shoot(self):
         bullet = Projectile(
             bullet_appearance,
             self.sprite_lists,
+            [],
```

Comment:
Passing empty parameters is often an indication the signature of function should be improved.

Reply:
Same as  above, most ordnance will have something in this list once hit markers are implimented